packages/core/src/App/Components/Elements/NotificationMessage/notification.jsx

```diff
@@ -157,13 +157,24 @@ Notification.propTypes = {
             route: PropTypes.string,
             text: PropTypes.string,
         }),
+        cta_btn: PropTypes.object,
+        className: PropTypes.string,
         closeOnClick: PropTypes.func,
         delay: PropTypes.number,
         header: PropTypes.string,
+        header_popup: PropTypes.string,
+        img_alt: PropTypes.string,
+        img_src: PropTypes.string,
         is_auto_close: PropTypes.bool,
+        // key: PropTypes.string,
```

Comment:
please remove this commented out line

Reply:
What are the problems with this line? Is it no longer needed and can be removed? If so it can be commented before removing.